From ed8694323669461e9a267c60b6156541336f2e7a Mon Sep 17 00:00:00 2001
From: Felix <google@ix5.org>
Date: Thu, 6 Jun 2019 00:06:45 +0200
Subject: [PATCH 2/3] vimrc.android: Set nomodeline

For security reasons.

Modeline functionality has been a constant source of security issues.

Latest:
https://github.com/numirias/security/blob/master/doc/2019-06-04_ace-vim-neovim.md

Signed-off-by: Felix <google@ix5.org>
Change-Id: Id6e680e7d2b29841717e63ee7668434a9d8466c4
---
 vimrc.android | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/vimrc.android b/vimrc.android
index 3c95fdb4b..2f7448828 100644
--- a/vimrc.android
+++ b/vimrc.android
@@ -15,6 +15,9 @@ set nocompatible
 " allow backspacing over everything in insert mode
 set backspace=indent,eol,start
 
+" Disable modeline parsing for security reasons
+set nomodeline
+
 " swap directories
 set directory=.,/data/local/tmp,/tmp
 
-- 
2.17.1

